dataland-keycloak-adapter/src/main/kotlin/org/dataland/keycloakAdapter/config/WebSecurityConfig.kt

```diff
@@ -74,17 +75,24 @@ class WebSecurityConfig(
 
     @Suppress("SpreadOperator")
     private fun authorizePublicLinksAndAddJwtConverter(http: HttpSecurity) {
-        val publicLinkMatchers = publicLinks.split(",").map { antMatcher(it) }.toTypedArray()
+        val linksList = listStringToList(publicLinks) + listStringToList(internalLinks)
+        val linkMatchers = linksList.map { antMatcher(it) }.toTypedArray()
         http
             .authorizeHttpRequests()
-            .requestMatchers(*publicLinkMatchers).permitAll()
+            .requestMatchers(*linkMatchers).permitAll()
             .anyRequest().fullyAuthenticated()
             .and()
             .logout().disable()
             .csrf().disable()
             .oauth2ResourceServer().authenticationManagerResolver(tokenAuthenticationManagerResolver())
     }
 
+    private fun listStringToList(listString: String) = if (listString.isNotEmpty()) {
+        listString.split(",")
+    } else {
+        emptyList()
+    }
```

Comment:
is this really necessary? if listString is empty, istn't `listString.split(",") == emptyList()`?

Reply:
sadly it holds that `"".split(",") == listOf("")`

Reply:
ok